src/Hintman/Config.hs

```diff
@@ -0,0 +1,43 @@
+module Hintman.Config
+    ( HintmanConfig(..)
+    , defaultHintmanConfig
+    , loadFileConfig
+    ) where
+
+import Toml (AnyValue (..), BiMap (..), BiToml, (.=))
+
+import qualified Toml
+
+
+
+data Check = TrailingSpaces
+           | TrailingNewlines deriving (Eq, Read, Show)
+
+newtype HintmanConfig = HintmanConfig
+    { hintmanConfigChecks :: [Check]
+    } deriving (Show)
+
+defaultHintmanConfig :: HintmanConfig
+defaultHintmanConfig = HintmanConfig [TrailingSpaces, TrailingNewlines]
+
+showCheck :: Check -> Text
+showCheck TrailingSpaces   = "trailing-spaces"
+showCheck TrailingNewlines = "trailing-newlines"
+
+parseCheck :: Text -> Maybe Check
```

Comment:
Instead of writing `parseCheck` function manually, you should use `inverseMap` from `relude`:

* http://hackage.haskell.org/package/relude-0.3.0/docs/Relude-Extra-Enum.html#v:inverseMap

Reply:
Wow, cool!